src/hooks/useWebData.ts

```diff
@@ -63,7 +63,7 @@ export default function useWebData() {
         accountAddress,
         {
           accountColor: colors.avatarBackgrounds[accountColor],
-          accountSymbol: wipeNotEmoji(accountSymbol),
+          accountSymbol: wipeNotEmoji(accountSymbol as string),
```

Comment:
I checked where `accountSymbol` comes from. If I am not mistaken it is `src/helpers/accountInfo.ts`. Then it can be a `string` or `false`:

```ts
const accountSymbol = returnStringFirstEmoji(
    emojiAvatar || addressHashedEmoji(accountAddress)
  );
```

So potentially it's risky to claim it's always `string`.